packages/muelu/src/Interface/MueLu_ParameterListInterpreter_def.hpp

```diff
@@ -1671,14 +1671,17 @@ namespace MueLu {
   // =====================================================================================================
   template <class Scalar, class LocalOrdinal, class GlobalOrdinal, class Node>
   void ParameterListInterpreter<Scalar, LocalOrdinal, GlobalOrdinal, Node>::
-  UpdateFactoryManager_Matlab(ParameterList& paramList, const ParameterList& defaultList, FactoryManager& manager,
-                              int levelID, std::vector<keep_pair>& keeps) const {
+  UpdateFactoryManager_Matlab(ParameterList& paramList, const ParameterList& /* defaultList */, FactoryManager& manager,
+                              int /* levelID */, std::vector<keep_pair>& /* keeps */) const {
 #ifdef HAVE_MUELU_MATLAB
     ParameterList Pparams = paramList.sublist("transfer: params");
     auto P = rcp(new TwoLevelMatlabFactory());
     P->SetParameterList(Pparams);
     P->SetFactory("P", manager.GetFactory("Ptent"));
     manager.SetFactory("P", P);
+#else
+    (void)paramList;
```

Comment:
MueLu developers should revisit this and have functions like this report an error if Matlab is not enabled.  It's fine just to disable the warnings for now.